architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,17 +175,23 @@ endif::[]
 [[image-streams]]
 == Image Streams
 
-An _image stream_ comprises any number of
-xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker-formatted
-container images] identified by tags. It presents a single virtual view of
-related images, similar to an image repository, and may contain images from any
-of the following:
+//from https://blog.openshift.com/image-streams-faq/
+An Image Stream is a pointer to an image using various identifying qualities. This means that even if the source image changes, 
+the Image Stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
 
-. Its own image repository in {product-title}'s
-xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[integrated
+//from image-streams-faq/
+An Image Stream contains all of the metadata information about any given image that is specified in the Image Stream specification. 
```

Comment:
this sentence seems redundant, all it says is "an imagestream contains the information that is specified in the imagestream"

I think it would be more useful to combine this w/ the next sentence and say "An imagestream does not contain actual image data, it is simply a mapping between tags and the actual images.  It can contain additional metadata that describes the tags provided by the imagestream."